### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by tomyan**

### CODE REVIEW

**Comment by tomyan** [comment]
@ivoreis docs still have a long way to go, but if you could sanity check how it's using docz and then I'll merge and get them publishing on release...

File: docs/src/index.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+name: Getting Started
+---
```
Comment (ivoreis) [inline docs/src/index.md]:
```suggestion
route: /
---
```

Comment (ivoreis) [reply docs/src/index.md]: We can define the page route too, have a look if this one works (if it doesn't work after deploying we might need to take the base path into account - `/opensource/cdflow2/`)

File: docs/src/cdflow-yaml.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
+---
+name: cdflow.yaml Reference
+---
```
Comment (ivoreis) [inline docs/src/cdflow-yaml.md]:
```suggestion
route: /cdflow
---
```

**Pull request merged**